Description: If ENOIOCTLCMD is not defined, then do so - fixes FTBFS.
Origin: Cherry picked from upstream 2.88 release
Reviewed-by: Dave Walker (Daviey) <DaveWalker@ubuntu.com>
Last-Update: 2011-02-03

--- a/src/bootlogd.c
+++ b/src/bootlogd.c
@@ -254,6 +263,9 @@
 	}
 
 #ifdef TIOCGDEV
+# ifndef  ENOIOCTLCMD
+#  define ENOIOCTLCMD	515
+# endif
 	if (ioctl(0, TIOCGDEV, &kdev) == 0) {
 		int r = findtty(res, "/dev", rlen, (dev_t)kdev);
 		if (0 != r)

